typings/css-properties.d.ts

```diff
@@ -1926,7 +1926,6 @@ export interface CSSPropertiesLossy {
     | undefined
     | Array<CSSPropertiesComplete[keyof CSSPropertiesComplete]>
     | CSSPropertiesLossy
-    | React.ReactChild
```

Comment:
This will cause the same problem of https://github.com/paypal/glamorous/issues/353

Reply:
This is minimal PoC I'm able to imagine.
https://goo.gl/kjc8XD

Reply:
IMO, to remove this extra union, we need to solve #353 first.